interfaces/ev_board_support.yaml

```diff
@@ -0,0 +1,58 @@
+description: This defines the board support package for the EV side
+cmds:
+  enable:
+    description: Enable/disable the simulation
+    arguments:
+      value:
+        description: true to enable, false to disable
+        type: boolean
+  set_cp_state:
+    description: Sets the CP State that should be set by the EV Yeti driver (controlled by S2)
```

Comment:
Should probably just say "EV board support driver" instead of "Yeti"

Reply:
Done :+1: